### ISSUE TITLE
comments should not accept html

### ISSUE THREAD

**Issue by markkap** [calmPress/calmpress#39]
instead probably some sort of markdown subset will be a better alternative. Right now seems like
** for bolding
> for quote
should cover most use cases.

This will be converted to HTML on submission. On the admin side editing full HTML should be retained.

======================

A better alternative, let the admin control the type of comments acceptable at the site, html (current), plain text which has no formatting and markdown for which there should be an editting toolbar.

**Comment by mikeschinkel** [comment]
Yes for a simple markdown.  No for no formatting at all.  And double-yes for a formatted vs. markup view with toolbar buttons for bolding, etc.
